deploy/crds/cloudingress.managed.openshift.io_apischemes_crd.yaml

```diff
@@ -62,7 +67,50 @@ spec:
                 code after modifying this file Add custom validation using kubebuilder
                 tags: https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html'
               type: string
+            conditions:
```

Comment:
how are these generated? I don't see any modification in the _types.go file to reflect this generation. Was it in another PR?

Reply:
Yes. This was a result of not running `operator-sdk generate ...` commands a long time ago after the status/conditions PR.